pallets/prices/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "pallet-prices"
+authors = ['Parallel Team']
+edition = '2018'
+version = '3.0.0'
```

Comment:
let's change version to 1.0.0 like in #80

Reply:
Fixed